src/app/modules/project/overview/overview.component.scss

```diff
@@ -152,3 +152,13 @@
     padding: 10px;
   }
 }
+
+accordion-group .card-header {
+  background: red!important;
```

Comment:
We really shouldn't use `!important`. Can this be overridden?

Reply:
This is all going to be fixed in "Fix: Accordion styling #106"